docs/content/direct/windows-setup-guide.md

```diff
@@ -0,0 +1,134 @@
+
+##  KubeStellar Installation Guide for Windows (PowerShell/CMD)
+
+## Work in Progress -- More steps coming soon!!!!!!
+
+### Prerequisites
+
+You need to install the following tools:
+
+1. [Git for Windows](https://git-scm.com/)
+2. [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+3. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/)
+4. [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+5. [Helm](https://helm.sh/docs/intro/install/)
+6. [OCM CLI (`clusteradm`)](https://open-cluster-management.io/)
+7. [kubeflex CLI (`kflex`)](https://github.com/kubestellar/kubeflex)
+
+
```

Comment:
The rendered content has two inconsistent numberings.

<img width="336" alt="image" src="https://github.com/user-attachments/assets/c83e80cf-4a8c-446c-8838-c9675160ebfe" />

<img width="198" alt="image" src="https://github.com/user-attachments/assets/fdcfa88b-4803-464d-85bc-5a07710edeca" />